conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/IntentResponse.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.conversation.v1.model;
+
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * IntentResponse.
+ */
+public class IntentResponse extends GenericModel {
+
+  /** The name of the intent. */
+  private String intent;
+  /** The description of the intent. */
+  private String description;
+  /** The timestamp for creation of the intent. */
+  private String created;
```

Comment:
Sadly, the swagger says that they are string.  I'll try modifying the swagger and see if the generator will produce the right thing.